### ISSUE TITLE
DStar users via crossmode to YSF are being blocked

### ISSUE THREAD

**Issue by PP5PK** [iu5jae/pYSFReflector#18]
Hello Daniel PU5KOD from Brazil here.

Just now I noticed a specific situation occurring with the current version of the reflector.

I have two active YSF reflectors and in both I was able to reproduce the fault, the situation is as follows, DStar users using OpenSpot3 doing crossmode to YSF are not able to pass the audio to the reflector, in the opposite way it can hear, I have been watching the log and noticed that these transmissions made in DStar are being blocked by some part of the rule, it is also worth remembering that I have no registered lock.
Below is a part of the log where a transmission made in DStar is blocked, user PR7DD is in DStar via crossmode to YSF using a Kenwood THD74.

M: 2021-06-28 14:53:09.168 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 14:53:09.168 Received end of transmission
M: 2021-06-28 14:53:17.952 Data from PR7DD   TH at PR7DD      blocked/CS
M: 2021-06-28 14:53:32.572 Removed from blockeds queue 32
M: 2021-06-28 14:53:44.280 Data from PR7DD   TH at PR7DD      blocked/CS
M: 2021-06-28 14:53:47.919 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 14:53:49.903 Removed from blockeds queue 32
M: 2021-06-28 14:54:48.367 Currently linked repeaters/gateways:
M: 2021-06-28 14:54:48.367      PR7DD     : 170.244.197.254:12082 3/60 : 2021-06-28 14:52:59
M: 2021-06-28 14:54:48.367      PU5KOD    : 187.108.0.156:42001 4/60 : 2021-06-28 14:53:09
M: 2021-06-28 14:54:48.367 No repeaters/gateways muted
M: 2021-06-28 14:55:27.889 Received end of transmission
M: 2021-06-28 14:55:33.505 Data from PR7DD   TH at PR7DD      blocked/CS
M: 2021-06-28 14:55:36.210 Removed from blockeds queue 32
M: 2021-06-28 14:55:39.939 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 14:55:58.810 Received end of transmission
M: 2021-06-28 14:56:02.620 Data from PR7DD   TH at PR7DD      blocked/CS
M: 2021-06-28 14:56:12.081 Removed from blockeds queue 32
M: 2021-06-28 14:56:18.489 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 14:56:26.653 Received end of transmission
M: 2021-06-28 14:56:35.233 Data from PR7DD   TH at PR7DD      blocked/CS
M: 2021-06-28 14:56:41.965 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 14:56:43.842 Removed from blockeds queue 32
M: 2021-06-28 14:56:47.538 Received end of transmission
M: 2021-06-28 14:56:48.370 Currently linked repeaters/gateways:
M: 2021-06-28 14:56:48.370      PR7DD     : 170.244.197.254:12082 2/60 : 2021-06-28 14:52:59
M: 2021-06-28 14:56:48.371      PU5KOD    : 187.108.0.156:42001 4/60 : 2021-06-28 14:53:09
M: 2021-06-28 14:56:48.371 No repeaters/gateways muted
M: 2021-06-28 14:58:48.467 Currently linked repeaters/gateways:
M: 2021-06-28 14:58:48.467      PR7DD     : 170.244.197.254:12082 4/60 : 2021-06-28 14:52:59
M: 2021-06-28 14:58:48.468      PU5KOD    : 187.108.0.156:42001 4/60 : 2021-06-28 14:53:09
M: 2021-06-28 14:58:48.468 No repeaters/gateways muted
M: 2021-06-28 15:00:35.068 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 15:00:48.568 Currently linked repeaters/gateways:
M: 2021-06-28 15:00:48.569      PR7DD     : 170.244.197.254:12082 1/60 : 2021-06-28 14:52:59
M: 2021-06-28 15:00:48.569      PU5KOD    : 187.108.0.156:42001 4/60 : 2021-06-28 14:53:09
M: 2021-06-28 15:00:48.569 No repeaters/gateways muted
M: 2021-06-28 15:01:21.942 Received end of transmission
M: 2021-06-28 15:02:48.638 Currently linked repeaters/gateways:
M: 2021-06-28 15:02:48.638      PR7DD     : 170.244.197.254:12082 3/60 : 2021-06-28 14:52:59
M: 2021-06-28 15:02:48.639      PU5KOD    : 187.108.0.156:42001 4/60 : 2021-06-28 14:53:09
M: 2021-06-28 15:02:48.639 No repeaters/gateways muted

I already inserted the callsign that is being blocked in the white list but I haven't had the opportunity to test it yet, but even if it works I would only solve this user's problem, leaving all others at the same circumstances blocked.

If anyone can check or give me any suggestions to resolve the situation I would be extremely grateful.

Thanks for listening!

**Comment by iu5jae** [comment]
The callsign PR7DD TH is not valid, remove "TH" or separate with - or / (PR7DD/TH or PR7DD-TH)

**Comment by PP5PK** [comment]
The "TH" is not part of the calling, I believe it is some information that the radio automatically inserts.
I asked another colleague who has an IC7100 to do the same and it was also blocked, in both cases the callsigns do not have a suffix.
Below is another part of the log where this user was also blocked, he is connected with two radios, a Yaesu C4FM and an ICom 7100, only Yaesu passes the transmission.

M: 2021-06-28 15:16:49.054      PR7DD     : 170.244.197.254:12082 5/60 : 2021-06-28 14:52:59
M: 2021-06-28 15:16:49.054      PU5KOD    : 187.108.0.156:42001 2/60 : 2021-06-28 14:53:09
M: 2021-06-28 15:16:49.055      M0SVU     : 88.104.155.254:42000 0/60 : 2021-06-28 15:03:21
M: 2021-06-28 15:16:49.055      M0SVU     : 88.104.155.254:53268 4/60 : 2021-06-28 15:10:32
M: 2021-06-28 15:16:49.055 No repeaters/gateways muted
M: 2021-06-28 15:17:25.184 Data from M0SVU   71 at M0SVU      blocked/CS
M: 2021-06-28 15:17:28.921 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 15:17:30.594 Received end of transmission
M: 2021-06-28 15:17:30.858 Removed from blockeds queue 36
M: 2021-06-28 15:17:36.420 Data from M0SVU   71 at M0SVU      blocked/CS
M: 2021-06-28 15:17:38.977 Removed from blockeds queue 36
M: 2021-06-28 15:17:44.851 Received data from M0SVU      to ALL        at M0SVU     
M: 2021-06-28 15:18:09.927 Received end of transmission
M: 2021-06-28 15:18:20.768 Data from M0SVU   71 at M0SVU      blocked/CS
M: 2021-06-28 15:18:32.487 Removed from blockeds queue 36
M: 2021-06-28 15:18:48.933 Data from M0SVU   71 at M0SVU      blocked/CS
M: 2021-06-28 15:18:49.155 Currently linked repeaters/gateways:
M: 2021-06-28 15:18:49.156      PR7DD     : 170.244.197.254:12082 1/60 : 2021-06-28 14:52:59
M: 2021-06-28 15:18:49.156      PU5KOD    : 187.108.0.156:42001 2/60 : 2021-06-28 14:53:09
M: 2021-06-28 15:18:49.156      M0SVU     : 88.104.155.254:42000 0/60 : 2021-06-28 15:03:21
M: 2021-06-28 15:18:49.157      M0SVU     : 88.104.155.254:53268 4/60 : 2021-06-28 15:10:32
M: 2021-06-28 15:18:49.157 No repeaters/gateways muted
M: 2021-06-28 15:19:11.165 Removed from blockeds queue 36
M: 2021-06-28 15:19:13.679 Received data from M0SVU      to ALL        at M0SVU     
M: 2021-06-28 15:19:26.860 Received end of transmission
M: 2021-06-28 15:19:29.779 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 15:20:36.549 Received end of transmission
M: 2021-06-28 15:20:41.880 Received data from M0SVU      to ALL        at M0SVU     
M: 2021-06-28 15:20:49.162 Currently linked repeaters/gateways:
M: 2021-06-28 15:20:49.162      PR7DD     : 170.244.197.254:12082 0/60 : 2021-06-28 14:52:59
M: 2021-06-28 15:20:49.162      PU5KOD    : 187.108.0.156:42001 2/60 : 2021-06-28 14:53:09
M: 2021-06-28 15:20:49.162      M0SVU     : 88.104.155.254:42000 0/60 : 2021-06-28 15:03:21
M: 2021-06-28 15:20:49.163      M0SVU     : 88.104.155.254:53268 4/60 : 2021-06-28 15:10:32
M: 2021-06-28 15:20:49.163 No repeaters/gateways muted
M: 2021-06-28 15:21:16.368 Received end of transmission
M: 2021-06-28 15:21:19.153 Received data from PU5KOD     to ALL        at PU5KOD    
M: 2021-06-28 15:22:25.827 Received end of transmission
M: 2021-06-28 15:22:31.999 Received data from M0SVU      to ALL        at M0SVU     
M: 2021-06-28 15:22:46.172 Received end of transmission

**Comment by PP5PK** [comment]
In this second case I noticed that it has the suffix "71" which is also not part of the call, I believe it is something that the radio automatically inserts

**Comment by PP5PK** [comment]
Any suggestions to solve the problem? Because the suffix being attached to the callsign cannot be removed manually and is being inserted automatically by the radio regardless of the user's will.
My concern is that everyone trying to use these YSF reflectors here using DStar to YSF crossmode is being blocked and they have no idea about it.

**Comment by dg9vh** [comment]
I think you can solve your problem by setting CheckRE=0 in your ini-file.

**Comment by PP5PK** [comment]
I'm running some tests here and I've already made some progress.
I found that when setting the callsign on a DStar radio, a "/" appears by default suggesting that it is necessary to insert a suffix, which practically all users do.
So that I don't have to set CheckRE to 0 and miss out on the benefits that callsign verification offers, I suggest that this syntax is accepted by default so that DStar radio operators are not caught in the rule and are automatically left out when blocked.
But thank you so much for your help, it was a great value!!

![IMG_20210628_163442.jpg](https://user-images.githubusercontent.com/3116740/123693765-d9da7500-d82e-11eb-960b-451f0d5f7ac1.jpg)
